dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java

```diff
@@ -752,4 +754,35 @@ private void createDummyFile(File testFile, int length) throws Exception {
     mWorker.completeFile(testFile.getPath(), CompleteFilePOptions.getDefaultInstance(),
         handle.getUUID().toString());
   }
+
+  @Test
+  public void testExists() throws Exception {
+    File rootFolder = mTestFolder.newFolder("root");
+    String rootPath = rootFolder.getAbsolutePath();
+    assertTrue(mWorker.exists(rootPath, ExistsPOptions.getDefaultInstance()));
+    String fileContent = "foobar";
+    File f = mTestFolder.newFile("root/f");
+    Files.write(f.toPath(), fileContent.getBytes());
+    assertTrue(mWorker.exists(f.getAbsolutePath(), ExistsPOptions.getDefaultInstance()));
+    mWorker.delete(f.getAbsolutePath(), DeletePOptions.getDefaultInstance());
+    assertFalse(mWorker.exists(f.getAbsolutePath(), ExistsPOptions.getDefaultInstance()));
+    mWorker.delete(rootPath, DeletePOptions.getDefaultInstance());
+    assertFalse(mWorker.exists(rootPath, ExistsPOptions.getDefaultInstance()));
+  }
+
+  @Test
+  public void testRename() throws IOException, AccessControlException {
+    File srcFolder = mTestFolder.newFolder("root");
+    String rootPath = srcFolder.getAbsolutePath();
+    mWorker.rename(rootPath, rootPath + "2", RenamePOptions.getDefaultInstance());
```

Comment:
can you use PathUtils.concatPath() instead?

Reply:
Ok,I wil